app/ModelSerializers/Summit/Presentation/PresentationTrackChairRatingTypeSerializer.php

```diff
@@ -0,0 +1,27 @@
+<?php namespace ModelSerializers;
+/**
+ * Copyright 2022 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+/**
+ * Class PresentationTrackChairRatingTypeSerializer
+ * @package ModelSerializers
+ */
+final class PresentationTrackChairRatingTypeSerializer extends SilverStripeSerializer
+{
+    protected static $array_mappings = [
```

Comment:
this serializer is missing collection score_types

please use this trait Many2OneExpandSerializer ( ie https://github.com/OpenStackweb/summit-api/blob/eacdd5ff4655dfb7ba68b0d664d7fa2af0024c88/app/ModelSerializers/ExtraQuestionTypeSerializer.php#L52)

Reply:
done